packages/eslint-plugin/src/util/misc.ts

```diff
@@ -160,7 +160,7 @@ type RequireKeys<
 > = ExcludeKeys<Obj, Keys> & { [k in Keys]-?: Exclude<Obj[k], undefined> };
 
 function getEnumNames<T extends string>(myEnum: Record<T, unknown>): T[] {
-  return Object.keys(myEnum).filter(x => isNaN(parseInt(x))) as T[];
+  return Object.keys(myEnum).filter(x => isNaN(+x)) as T[];
```

Comment:
Why wouldn't we use this?
```suggestion
  return Object.keys(myEnum).filter(x => isNaN(parseInt(x, 10))) as T[];
```

Reply:
I personally prefer using an operator instead of a function when available, especially since the `+` operator works more safely than `parseInt` does by default. However, I'm happy to change it if you'd like

Reply:
Gotcha, I'm cool with it, just wanted to check to make sure there was intentionality 👍 . I can never quite keep straight the differences between `parseInt()` and `Number()`/`+` but I trust and defer to your judgement here 🙂.

I would personally say I do much prefer the explicit style....
```suggestion
  return Object.keys(myEnum).filter(x => isNaN(Number(x))) as T[];
```
Up to you though!

Reply:
I also prefer `Number(x)` :)